.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "prettier.useEditorConfig": false,
+    "prettier.singleQuote": true,
+    "prettier.semi": false,
+    "prettier.printWidth": 120
```

Comment:
"prettier.trailingComma": "es5",
"prettier.tabWidth": 2,
"prettier.semi": false,
"prettier.singleQuote": true,
"prettier.printWidth": 120,
"prettier.arrowParens": "always",

Reply:
It's important to establish the tabWidth so all files have the same indentation.

Reply:
done!